test/libsolidity/syntaxTests/constants/redefinition_public_constant.sol

```diff
@@ -0,0 +1,4 @@
+uint constant public c = 7;
```

Comment:
Shouldn't this already be an error due to the ``public``? Or what does ``public`` or non-``public`` mean for constants?